.github/labeler.yml

```diff
@@ -54,3 +54,9 @@
 - aten/src/ATen/quantized/**
 - aten/src/ATen/native/quantized/cpu/**
 - test/quantization/**
+
+"ciflow/binaries_wheel":
+- .ci/docker/ci_commit_pins/triton.txt
+
+"ciflow/binary_conda":
```

Comment:
Also can remove the latter here

Reply:
> Not really needed, single manywheel binary job that exists on trunk should be sufficient


> > Not really needed, single manywheel binary job that exists on trunk should be sufficient
> 
> Oh, I get it now. the PR didn't even have `ciflow/trunk` at first. Thus, no signal.
> 
> Having a rule to add `ciflow/trunk` will help though in case the change was forced merge right off the bat.


Hmmm...I'm starting to really feel like we need to tweak mergebot somehow to make sure users realize when they're skipping `trunk` tests, to make sure it's a deliberate choice

Reply:
Yeah, this has been seen many times recently.  I kind of miss the previous behavior of adding trunk right away after approval.  Let's chat more on this with the team on how we can avoid this.